pkgs/top-level/all-packages.nix

```diff
@@ -13724,6 +13716,20 @@ in
 
   lyx = callPackage ../applications/misc/lyx { };
 
+
+  magnetophonDSP = {
```

Comment:
why this name? Is this set only intended for you? In that case, why should it be in Nixpkgs? Or is this just to somewhat order these packages? In that case a `dsp` set may make sense. Or, if these are only `lv2` plugins, maybe `lv2packages`?